dev/storage_base.h

```diff
@@ -229,11 +231,11 @@ namespace sqlite_orm {
                     },
                     /* call = */
                     [](sqlite3_context* context, void* functionVoidPointer, int argsCount, sqlite3_value** values) {
-                        auto& functionPointer = *static_cast<F*>(functionVoidPointer);
+                        auto& fn = *static_cast<F*>(functionVoidPointer);
```

Comment:
better make less acronyms to make code more readable for newbies and us

Reply:
So, leave `functionPointer`?

Reply:
yep